### ISSUE TITLE
Border not visible on Safari

### ISSUE THREAD

**Issue by seopei** [stephanrauh/ngx-extended-pdf-viewer#1732]
**Describe the bug**
On Safari only the first page has a border.

**Version info**
Angular 15.2.0, pdf.js 3.4.492, and ngx-extended-pdf-viewer 16.2.0

**Desktop (please complete the following information):**
 - Browser: Safari Version 16.3 (18614.4.6.1.6)

**To Reproduce**
https://pdfviewer.net/extended-pdf-viewer/simple with vertical scrolling

**Screenshots**
![Screenshot 2023-03-30 at 16 44 58](https://user-images.githubusercontent.com/22980478/228876035-fd5657ec-dced-46c3-a9a9-83d54b9b3f55.png)

**Solution**
```
  ngx-extended-pdf-viewer .pdfViewer .page {
      filter: opacity(1);
      -webkit-filter: opacity(1);
    }
```

**Comment by stephanrauh** [comment]
Such a surprising bug, and the solution is even more surprising! It works - but I'd like to know why. Can you explain to me what goes wrong, and why the `-webkit-filter` solves the problem?

**Comment by seopei** [comment]
I don't really know why, but I had the same bug once with an [other repo](https://github.com/VadimDez/ng2-pdf-viewer/pull/509/files).
So I found [this](https://stackoverflow.com/questions/64700115/why-iphone-safari-breaks-shadows-on-elements-in-pdf) and [this](https://stackoverflow.com/questions/21025212/border-image-not-showing-in-safari) on stackoverflow ;) 
It is possible that a other solution instead of `filter` will work: `border: 9px solid rgba(0,0,0,0.01);` instead of `border: 9px solid transparent;`

**Comment by stephanrauh** [comment]
Your bugfix has landed with version 16.2.16.

Enjoy!
Stephan
